internal/chezmoi/sourcestate.go

```diff
@@ -202,6 +210,7 @@ func NewSourceState(options ...SourceStateOption) *SourceState {
 		umask:                Umask,
 		encryption:           NoEncryption{},
 		ignore:               newPatternSet(),
+		httpClient:           http.DefaultClient,
 		logger:               &log.Logger,
 		readTemplateData:     true,
 		priorityTemplateData: make(map[string]interface{}),
```

Comment:
@twpayne I'm not sure it's the right place for `httpClient`

Reply:
Nor am I, but it's a perfectly reasonable place. The main reason I tend to order things alphabetically is that it means I no longer have to make a decision about where to add something, or which way to scroll to find something. In `struct`s though, where I generally want to keep related fields together, it's less clear.